src/vos/vos_layout.h

```diff
@@ -105,7 +105,7 @@ enum vos_gc_type {
 #define POOL_DF_MAGIC				0x5ca1ab1e
 
 #define POOL_DF_VER_1				1
-#define POOL_DF_VERSION				3
+#define POOL_DF_VERSION				4
```

Comment:
might just be me, but tabs look off

Reply:
I think it's just github displaying it wrong